presto-product-tests/src/main/java/io/prestosql/tests/hive/TestAvroSymlinkInputFormat.java

```diff
@@ -87,12 +74,56 @@ public void testSymlinkTable()
                 "STORED AS " +
                 "INPUTFORMAT 'org.apache.hadoop.hive.ql.io.SymlinkTextInputFormat' " +
                 "OUTPUTFORMAT 'org.apache.hadoop.hive.ql.io.HiveIgnoreKeyTextOutputFormat'");
-
+        hdfsClient.createDirectory(warehouseDirectory + "/TestAvroSymlinkInputFormat/data");
```

Comment:
remove?

Reply:
removed by `hdfsClient.delete(warehouseDirectory + dataRoot);`. I reorganized to make this easier to follow.

Reply:
This is removed by `hdfsClient.delete(warehouseDirectory + dataRoot);`. I refactored slightly to make this explicit.